README.md

```diff
@@ -76,227 +43,71 @@ Example `Vagrantfile` for Windows can be found [here](https://gist.github.com/st
-Here's an example list of `sudoers`:
-
-```yaml
-sudoers:
-  - user: "admin"
-    groups: ["sudo"]
-  - user: "another_user"
-    groups: ["sudo"]
-```
-
-`user` is the key to be used when creating the user and `groups` is an array used to determine the groups the user belongs to. The first item in the array will be used when setting the user's primary group.
-
-Here's an example list of `sudoer_passwords`:
-
-```yaml
-sudoers:
-  admin: $6$rounds=100000$JUkj1d3hCa6uFp6R$3rZ8jImyCpTP40e4I5APx7SbBvDCM8fB6GP/IGOrsk/GEUTUhl1i/Q2JNOpj9ashLpkgaCxqMqbFKdZdmAh26/
-  another_user: $6$rounds=100000$r3ZZsk/uc31cAxQT$YHMkmKrwgXr3u1YgrSvg0wHZg5IM6MLEzqOraIXqh5o7aWshxD.QaNeCcUX3KInqzTqaqN3qzo9nvc/QI0M1C.
-```
-
-The passwords were generated using the python command [found here](http://docs.ansible.com/faq.html#how-do-i-generate-crypted-passwords-for-the-user-module). The passwords generated here are `example_password` and `another_password`, respectively. The ansible user module doesn't handle any encryption and passwords must be encrypted beforehand. It's also recommended `vars/sudoer_passwords.yml` be encrypted using one of the encryption methods described in the [passwords](#passwords) section of this document. Passwords are stored separately in order to ease the separation of encrypted var files and are looked up based on the user name.
-
-This playbook should be run on remote hosts and is designed to be run once whenever the server is initially created (as the root user will be inaccessible after running and the ansible default user is the [current user](http://docs.ansible.com/intro_configuration.html#remote-user)). To invoke, run `ansible-playbook -i hosts/ENVIRONMENT secure-root.yml`. Because the root user is locked down, remote hosts also need to use a different ssh user. You can set the default remote user in `site.yml` by setting the `remote_user` variable. All subsequent provisions need to have the `--ask-sudo-pass` flag set when using `ansible-playbook`.
-
-### `fail2ban` and `ferm`
-
-> Fail2ban scans log files (e.g. /var/log/apache/error_log) and bans IPs that show the malicious signs -- too many password failures, seeking for exploits, etc.
-
-From http://www.fail2ban.org/wiki/index.php/Main_Page
-
-> ferm is a tool to maintain complex firewalls, without having the trouble to rewrite the complex rules over and over again. ferm allows the entire firewall rule set to be stored in a separate file, and to be loaded with one command.
-
-From http://ferm.foo-projects.org/
-
-You can read the role documentation for `fail2ban` [here](roles/fail2ban/README.md) and `ferm` [here](roles/ferm/README.md).
-
-## Todo
-
-* Multisite: basic support is included but not yet complete. There are issues with doing a network install from scratch via WP-CLI.
-* Nginx: configuration needs more options and advanced setups like static files and subdomain multisite support.
+* [Locking down root](https://github.com/roots/bedrock-ansible/wiki/Security#locking-down-root)
```

Comment:
Should just expand this a little:

> The `secure-root.yml` playbook is provided to help secure your remote servers including better SSH security.
   See the Wiki for [Locking down root](https://github.com/roots/bedrock-ansible/wiki/Security#locking-down-root).